src/Monticello-Tests/MCWorkingCopyForExtensionsTest.class.st

```diff
@@ -0,0 +1,43 @@
+Class {
+	#name : #MCWorkingCopyForExtensionsTest,
+	#superclass : #MCTestCase,
+	#category : #'Monticello-Tests-Base'
+}
+
+{ #category : #tests }
+MCWorkingCopyForExtensionsTest >> tearDown [
+	super tearDown.
```

Comment:
The super tearDown should be the last call of the tearDown

Reply:
Yes - there even is a lint rule for this